From e65ce78ff23400fd8090037d8bda7470df44d122 Mon Sep 17 00:00:00 2001
From: Xufeng Zhang <xufeng.zhang@windriver.com>
Date: Mon, 25 Jul 2011 19:03:15 +0800
Subject: [PATCH 30/47] 85xx/powerpc: l2sram: Add compatible entry for some FSL platforms

Add entry for P1012, P1024, P1015, mpc8572 etc in device ID table.

Extracted from vendor drop QorIQ-NONDPAA-SDK-V1-20110429_ltib.iso
0014-85xx-powerpc-l2sram-Add-compatible-entry-for-some-FS.patch.

Signed-off-by: Harninder Rai <harninder.rai@freescale.com>
Signed-off-by: Li Yang <leoli@freescale.com>
Integrated-by: Xufeng Zhang <xufeng.zhang@windriver.com>
---
 arch/powerpc/sysdev/fsl_85xx_l2ctlr.c |   28 ++++++++++++++++++++++++++--
 1 files changed, 26 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c b/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c
index 15cc380..85eef3e 100644
--- a/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c
+++ b/arch/powerpc/sysdev/fsl_85xx_l2ctlr.c
@@ -194,17 +194,41 @@ static struct of_device_id mpc85xx_l2ctlr_of_match[] = {
 		.compatible = "fsl,p1020-l2-cache-controller",
 	},
 	{
+		.compatible = "fsl,p1011-l2-cache-controller",
+	},
+	{
 		.compatible = "fsl,p1010-l2-cache-controller",
 	},
 	{
-		.compatible = "fsl,p1011-l2-cache-controller",
+		.compatible = "fsl,p1014-l2-cache-controller",
 	},
 	{
-		.compatible = "fsl,p1013-l2-cache-controller",
+		.compatible = "fsl,p1021-l2-cache-controller",
+	},
+	{
+		.compatible = "fsl,p1012-l2-cache-controller",
 	},
 	{
 		.compatible = "fsl,p1022-l2-cache-controller",
 	},
+	{
+		.compatible = "fsl,p1013-l2-cache-controller",
+	},
+	{
+		.compatible = "fsl,mpc8572-l2-cache-controller",
+	},
+	{
+		.compatible = "fsl,p1024-l2-cache-controller",
+	},
+	{
+		.compatible = "fsl,p1015-l2-cache-controller",
+	},
+	{
+		.compatible = "fsl,p1025-l2-cache-controller",
+	},
+	{
+		.compatible = "fsl,p1016-l2-cache-controller",
+	},
 	{},
 };
 
-- 
1.7.0.4

